tests/integration/adaptation-editor/manual_test_description_generated/list-report-v2.md

```diff
@@ -0,0 +1,363 @@
+# List Report V2 Test Documentation
+
+## Table of Contents
+
+- [1. Enable/Disable clear filter bar button](#1-enabledisable-clear-filter-bar-button)
+- [2. Add controller to page](#2-add-controller-to-page)
+- [3. Change table columns](#3-change-table-columns)
+- [4. Add Custom Table Action](#4-add-custom-table-action)
+- [5. Add Custom Table Column](#5-add-custom-table-column)
+- [6. Enable/Disable Semantic Date Range in Filter Bar](#6-enabledisable-semantic-date-range-in-filter-bar)
+- [7. Enable Variant Management in Tables and Charts](#7-enable-variant-management-in-tables-and-charts)
+- [8. Change table actions](#8-change-table-actions)
+- [Add New Annotation File](#add-new-annotation-file)
+
+<a id="1-enabledisable-clear-filter-bar-button"></a>
+## 1. Enable/Disable clear filter bar button
+
+### Steps
+
+1. page.goto(http://localhost:3000/adaptation-editor.html?fiori-tools-rta-mode=true#app-preview)
+2. Check that UIAdaptation mode is enabled
+3. Check `Clear` Button is hidden
+4. Click on button `Enable "Clear" Button in Filter Bar`
+5. Check `Clear` Button is visible
+6. Click on button `Save`
+7. Check `Save` button is disabled
+8. Verify changes:
+
+## Change(s)
```

Comment:
This probably doesn't need to be a title.

Reply:
adapted